gitlab/tests/test_e2e.py

```diff
@@ -17,6 +18,7 @@ def test_e2e_legacy(dd_agent_check):
     aggregator.assert_metrics_using_metadata(get_metadata_metrics())
 
 
+@mock.patch.dict('os.environ', {'DDEV_SKIP_GENERIC_TAGS_CHECK': 'true'})
```

Comment:
nit - you could also declare this in the tox file, I think this is what we do in most of the integrations

Reply:
Done